File: code/1e235.cpp
#include <iostream>
using namespace std;
int main()
{
    int i,n, a[1000],dmin;
    cin>>n;
    for (i=1; i<=n; i++) cin>>a[i];
    dmin=-1;
    for (i=1; i<=n; i++) 
        if (a[i]>0) {
            dmin=a[i];
            break;
        }
    if (dmin>0) {
        for (i=1; i<=n; i++)
            if (a[i]>0 && a[i]<dmin)
                dmin=a[i];
    }
    if (dmin>0) cout<<dmin;
    else cout<<"KHONG CO";
    return 0;
}

File: code/1e235.pas
var i, n, dmin: integer;
    a : array[1..1000] of integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    dmin:=-1;
    for i:= 1 to n do
        if a[i]>0 then begin
            dmin:=a[i];
            break;
        end;
    if dmin>0 then
        for i:= 1 to n do
            if (a[i]>0) and (a[i]<dmin) then
                dmin:=a[i];
    if dmin>0 then write(dmin)
    else write('KHONG CO');
END.